packages/sw-background-sync-queue/src/lib/broadcast-manager.js

```diff
@@ -1,17 +1,20 @@
-import {defaultBroadcastChannelName, broadcastMeta} from './constants';
+import {broadcastMeta} from './constants';
 import assert from '../../../../lib/assert';
 
-let bcmanager;
+/**
+ * broadcasts the message with the guven type and url
+ *
+ * @param {Object} config
+ */
+function broadcastMessage(broadcastManager, {type, url}) {
```

Comment:
These parameters look odd. Can you normalize things so that `broadcastManager` is take in as a named property in a single Object parameter, alongside `type` and `url`?